README.md

```diff
@@ -75,11 +75,30 @@ WebMock.disable_net_connect!(:net_http_connect_on_start => true, :allow_localhos
 ## Formatter modes
 
 The following modes are supported:
-
-| Name | Purpose |
-| --- | --- |
-| attach_to_launch | Do not create a new launch but add executing features/scenarios to an existing launch. Use launch_id or file_with_launch_id settings to configure that. If they are not present client will check rp_launch_id.tmp in `Dir.tmpdir`)
-| use_same_thread_for_reporting | Send reporting commands in the same main thread used for running tests. This mode is useful for debugging this Report Portal client. It changes default behavior to send commands in the separate thread. Default behavior is there not to slow test execution. |
+<table><thead><tr><th>Name</th><th>Purpose</th></tr></thead>
+<tbody>
+<tr>
+<td>attach_to_launch</td>
+<td>
+Add executing features/scenarios to an existing launch. 
+Use following options to configure that. 
+
+    1. launch_id
+    2. file_with_launch_id 
+    3. rp_launch_id.tmp in `Dir.tmpdir` 
+    
+   If above options not present client will create new launch
```

Comment:
An intention of `attach_to_launch` option is to prevent a launch being created automatically.
A launch should be created automatically only if `attach_to_launch` mode is not present.
Why do you disagree with this behavior?

Reply:
I'm not sure what is the use for `attach_to_launch` in your project, but If I need to connect to the ReportPortal from outside of the formatter in order to create a launch that it should attach to, it sounds more like a HACK! 
Also, it's redundant to provide `rp_launch` in this case and if launch created from by some other scripts. 
Same for `file_with_launch_id` - if I provide this option it should enable `attach_to_launch` by default .

Now as we are talking about different modes for formatter, parrallel_cucumber have some interesting behaviours with standard formatters, when executed with parallel_cucumber: 
- JSON or HTML formatter - generate report per process, you end up with a report that may have multiple feature files in a single report(if the number of processes lower than the number of features).
- ReportPortal::Cucumber::ParallelFormatter - if I remember correctly by default have the same behaviour. This is not practical considering the fact that the report name will be the same(but different content as feature files executed in different processes). 
*Report name it's used by REPORT PORTAL to group reports and to achieve good AI you need to have consistent runs, with same feature files and options. 
- ReportPortal::Cucumber::ParallelFormatter [`attach_to_launch`] - sounds like a good alternative 
  - default - create the new launch for each parallel_cucumber run and terminate it after. (most cucumber users should be happy with this option)
     -  launch_uuid - add the ability to create a custom lock file (useful if cucumber farm have shared temp folder)
  - launch_id / file_with_launch_id / rp_launch_id.tmp in `Dir.tmpdir` - this should attach to existing launch and it also does not terminate launch.  ( If you want to do more work outside of the test framework, have some farm setup or something)
 

not sure what is there to disagree with if you don't really have it working in the first place? I also don't understand the purpose of two formatters when single formatter should be able to do everything.